src/filesystem/impls/filer/FilerUtils.js

```diff
@@ -22,4 +22,11 @@ define(function (require, exports, module) {
             return path;
         }
     };
+
+    // Normalize '.html', 'html', '.HTML', 'HTML' all to '.html', unless exludePeriod is 'true'
+    // then make it just 'html' without the period.
+    exports.normalizeExtension = function(ext, exludePeriod) {
```

Comment:
`exludePeriod` should be `excludePeriod` (missing a `c`).  Can you please fix here and in the line below?